scripts/split-e2e-tests.ts

```diff
@@ -60,6 +60,10 @@ type CandidateJob = {
 const FORCE_REGION_MAP = {
   interactions: 'us-west-2',
   containers: 'us-east-1',
+  'rds-pg-userpool-auth-fields': 'ap-northeast-2',
+  'rds-pg-oidc-auth-fields': 'ap-northeast-2',
+  'rds-mysql-userpool-auth-fields': 'ap-northeast-2',
+  'rds-mysql-oidc-auth-fields': 'ap-northeast-2',
```

Comment:
we have a [task in our Gen-2 project](https://app.asana.com/0/1206048977824382/1206394910596509/f) to remove these overrides. Will take that up post merge.